lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -14,6 +14,9 @@
 # You should have received a copy of the GNU General Public License
 # along with Ansible. If not, see <http://www.gnu.org/licenses/>.
 
+# Make coding more python3-ish
+from __future__ import (absolute_import, division, print_function)
```

Comment:
I had a look at q;  it's fine for debugging and I'll probably use it somehow, but it seems that it could leave debugging information in a public file in /tmp which could be a security problem if it accidentally went into production so I'm not totally convinced by it and I won't use it in any case in code I push up to github until I better understand how to use it securely.